docs/wise_agents_architecture.md

```diff
@@ -1,28 +1,42 @@
-# Wise-agents architecture
+# Wise-Agent Architecture
 
-TODO: chart
+*Disclaimer: This page is under construction and continuously updated.*
 
-## LLM integration
+## General Architecture
 
-### OpenAI API
+WiseAgent is a [multi-agent intelligent framework](multi_agents.md) designed for distributed, asynchronous communication. It is capable of connecting to any LLM model or inference system that supports the OpenAI API, enabling the use of multiple different models within the same configuration. One of the key features of WiseAgent is its cloud readiness and distributed nature, allowing for the deployment of different components in separate pods, while also being able to run everything in a single process for simpler use cases.
 
-### Local vs remote models
+The following image provides a high-level overview of the architecture and flow:
 
-#### Ollama for local models
+![architecture](/images/architecture.jpg)
```

Comment:
Just adding a question about the architecture diagram here since I can't comment on it below. Just wanted to check what's meant by "Assistant Agent" vs. "Agent" in the diagram below? Is the distinction that an Assistant Agent can make use of tools?

Reply:
It's more an agent able to interact with the user